mvn site is fail after mybatis-parent 31-SNAPSHOT

Probably, this behavior occurred by updating mybatis-parent 31-SNAPSHOT.

```
...
[INFO] --- license-maven-plugin:3.0:format (default) @ mybatis ---
[INFO] 
[INFO] <<< clirr-maven-plugin:2.8:clirr < compile @ mybatis <<<
[INFO] 
[INFO] 'compile' forked phase execution for clirr-maven-plugin:clirr report preparation done
[INFO] configuring report plugin org.codehaus.mojo:versions-maven-plugin:2.5
[INFO] configuring report plugin org.jacoco:jacoco-maven-plugin:0.7.9
[INFO] Fork Value is true
     [java] Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 5
     [java]     at edu.umd.cs.findbugs.BugCounts.addError(PackageStats.java:51)
     [java]     at edu.umd.cs.findbugs.PackageStats.addError(PackageStats.java:240)
     [java]     at edu.umd.cs.findbugs.ProjectStats.addBug(ProjectStats.java:272)
     [java]     at edu.umd.cs.findbugs.SortedBugCollection.add(SortedBugCollection.java:956)
     [java]     at edu.umd.cs.findbugs.SortedBugCollection.add(SortedBugCollection.java:224)
     [java]     at edu.umd.cs.findbugs.BugCollectionBugReporter.doReportBug(BugCollectionBugReporter.java:99)
     [java]     at edu.umd.cs.findbugs.AbstractBugReporter.reportBug(AbstractBugReporter.java:175)
     [java]     at edu.umd.cs.findbugs.DelegatingBugReporter.reportBug(DelegatingBugReporter.java:69)
     [java]     at edu.umd.cs.findbugs.FilterBugReporter.reportBug(FilterBugReporter.java:49)
     [java]     at edu.umd.cs.findbugs.detect.UnreadFields.report(UnreadFields.java:1070)
     [java]     at edu.umd.cs.findbugs.DetectorToDetector2Adapter.finishPass(DetectorToDetector2Adapter.java:55)
     [java]     at edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1114)
     [java]     at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:280)
     [java]     at edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:401)
     [java]     at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1187)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 34.134 s
[INFO] Finished at: 2018-03-24T17:58:06+09:00
[INFO] Final Memory: 72M/662M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.6:site (default-site) on project mybatis: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.6:site failed: Java returned: 1 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

## Related Links

* #1207

--- a/pom.xml
+++ b/pom.xml
@@ -137,7 +137,7 @@
     <maven.compiler.testSource>1.8</maven.compiler.testSource>
     <maven.compiler.testCompilerArgument>-parameters</maven.compiler.testCompilerArgument>
     <clirr.comparisonVersion>3.3.1</clirr.comparisonVersion>
-    <findbugs.onlyAnalyze>org.apache.ibatis.*</findbugs.onlyAnalyze>
+    <spotbugs.onlyAnalyze>org.apache.ibatis.*</spotbugs.onlyAnalyze>
     <osgi.export>org.apache.ibatis.*;version=${project.version};-noimport:=true</osgi.export>
     <osgi.import>*;resolution:=optional</osgi.import>
     <osgi.dynamicImport>*</osgi.dynamicImport>

--- a/src/main/java/org/apache/ibatis/mapping/BoundSql.java
+++ b/src/main/java/org/apache/ibatis/mapping/BoundSql.java
@@ -27,8 +27,8 @@ import org.apache.ibatis.session.Configuration;
  * An actual SQL String got from an {@link SqlSource} after having processed any dynamic content.
  * The SQL may have SQL placeholders "?" and an list (ordered) of an parameter mappings 
  * with the additional information for each parameter (at least the property name of the input object to read 
- * the value from). 
- * </br>
+ * the value from).
+ * <p>
  * Can also have additional parameters that are created by the dynamic language (for loops, bind...).
  *
  * @author Clinton Begin

--- a/src/main/java/org/apache/ibatis/reflection/ParamNameResolver.java
+++ b/src/main/java/org/apache/ibatis/reflection/ParamNameResolver.java
@@ -105,8 +105,8 @@ public class ParamNameResolver {
 
   /**
    * <p>
-   * A single non-special parameter is returned without a name.<br />
-   * Multiple parameters are named using the naming rule.<br />
+   * A single non-special parameter is returned without a name.
+   * Multiple parameters are named using the naming rule.
    * In addition to the default names, this method also adds the generic names (param1, param2,
    * ...).
    * </p>

